include/bout/sys/timer.hxx

```diff
@@ -86,6 +86,8 @@ private:
     seconds time;                   ///< Total time
     bool running;                   ///< Is the timer currently running?
     clock_type::time_point started; ///< Start time
+    uint counter{0};                ///< Number of Timer objects associated with this
```

Comment:
To be really picky, `uint` is not a standard type -- please could you change this to `unsigned int`? :)